packages/devtools/.prettierrc.js

```diff
@@ -1,7 +1,7 @@
 // NOTE: This package has inconsistent use of trailing commas and bracket spacing
```

Comment:
Isn't it trivial to fix this with prettier? :P

Reply:
Removed :) Not relevant anymore